packages/frontend-shared/src/locales/en-US.json

```
@@ -237,6 +237,13 @@
     },
     "nav": {
       "ariaLabel": "Pages"
+    },
+    "debug": {
+      "new": "New",
+      "debugFeature": "New Debug feature",
+      "passed": "Last recorded run passed",
+      "failed": "Last recorded run had {n} test failure | Last recorded run had {n} test failures",
+      "errored": "Last recorded run had an error"
```

```suggestion
      "passed": "Relevant run passed",
      "failed": "Relevant run had {n} test failure | Relevant run had {n} test failures",
      "errored": "Relevant run had an error"
```
Nice addition to include these labels for the badge.  I do believe we have settled on on official term to call these runs, but "relevant" has been used in general conversation about the issue.  This term is meant to convey that the run is "relevant" to the current state of the code the developer currently has checked out from git.

---

Addressed in [4c21f5f](https://github.com/cypress-io/cypress/pull/25152/commits/4c21f5f4d005a19a9dbe52ffac3101f9bfeb513f)